SJH/Day5/음양더하기reduce.js

```diff
@@ -0,0 +1,13 @@
+function solution(absolutes, signs) {
+    const answer = absolutes.reduce((plus, ij, index) => {
+        const ji = signs[index];   
+        let realji;                
+        if (ji === true) {         
```

Comment:
```suggestion
        if (ji) {         
```

`ji`는 이미 그 자체로 `true`, `false`인 값이라서 `ji`만 작성하더라도 참인 조건을 나타냅니다. 더 깔끔하기도 해요!

추가적으로, 이렇게 작성하게 될 경우 변수가 담고있는 값만으로도 참인지 거짓인지 구분할 수 있습니다!

아래는 Falsy 값 목록... (전혀 외울필요 없어요! 😉)
```
false
0
-0
0n (BigInt 0)
"" (빈 문자열)
null
undefined
NaN
```